sys/arduino/serialport.cpp

```diff
@@ -92,7 +92,7 @@ size_t SerialPort::print(float val)
 size_t SerialPort::print(float val, int format)
 {
     char buf[64];
-    size_t len = sprintf(buf, "%.*f", format, val);
+    size_t len = sprintf(buf, "%.*f", format, (double)val);
```

Comment:
because otherwise the sprintf function raises this error:
```
serialport.cpp:95:50: error: format ‘%f’ expects argument of type ‘double’, but argument 4 has type ‘float’ [-Werror=format=]
```